README.md

```diff
@@ -0,0 +1,9 @@
+AclBundle
+=========
+
```

Comment:
As this is the main readme of the repo (not a subtree split), it would be great to have a bit more content in it